ds701_book/04-Linear-Algebra-Refresher.qmd

```diff
@@ -243,61 +217,112 @@ ax.grid()
 plt.show()
 ```
 
-::: {.content-hidden when-profile="slides"}
-#### Dot product
-:::
+---
 
-::: {.content-visible when-profile="slides"}
-## Dot product
-:::
+**Dot product:** Let $\mathbf{u},\mathbf{v}\in\mathbb{R}^{n}$ then the dot product is defined as
+$$
+\mathbf{u}\cdot\mathbf{v} = \sum_{i=0}^n u_i v_i.
+$$
+
+**Vector 2-norm:** The 2-norm of a vector $\mathbf{v}\in\mathbb{R}^{n}$ is defined as
+$$
+\Vert \mathbf{v}\Vert_2 = \sqrt{\mathbf{v}\cdot\mathbf{v}} = \sqrt{\sum_{i=1}^n v_i^2}.
+$$
+
+This norm is referred to as the $\ell_2$ norm. In these notes, the notation $\Vert \mathbf{v} \Vert$, indicates the 2-norm.
+
+---
+
+**Unit vector:** A unit vector $\mathbf{v}$ is a vector such that $\Vert \mathbf{v} \Vert_2 = 1$. 
+    - All vectors of the form $\frac{\mathbf{v}}{\Vert \mathbf{v} \Vert_2 }$ are unit vectors.
 
-The dot product of two vectors $\mathbf{u}, \mathbf{v}$ can be used to project $\mathbf{u}$ onto $\mathbf{v}$. This is illustrated in @fig-dot-product.
+**Distance:** Let $\mathbf{u},\mathbf{v}\in\mathbb{R}^{n}$, the distance between $\mathbf{u}$ and $\mathbf{v}$ is
+$$
+\Vert \mathbf{u} - \mathbf{v} \Vert_2.
+$$
+
+**Orthogonality:** Two vectors $\mathbf{u},\mathbf{v}\in\mathbb{R}^{n}$ are orthogonal if and only if $\mathbf{u}\cdot\mathbf{v}=0$. 
+
+**Angle between vectors:**  Let $\mathbf{u},\mathbf{v}\in\mathbb{R}^{n}$, the angle between these vectors is 
+$$
+\cos{\theta} = \frac{\mathbf{u}\cdot\mathbf{v}}{\Vert \mathbf{u}\Vert_2 \Vert\mathbf{v}\Vert_2}.
+$$
+
+---
+
+The dot product of two vectors $\mathbf{u}, \mathbf{v}$ can be used to project $\mathbf{u}$ onto $\mathbf{v}$. 
+
+$$
+\mathrm{proj}_{\mathbf{v}}\mathbf{u} = \frac{\mathbf{u}\cdot\mathbf{v}}{\Vert \mathbf{v} \Vert^2}\mathbf{v}
```

Comment:
I would move the period from the end of $\mathbf{v}$. to be inside the math equation. It's just how I was trained to write mathematics.